beets/autotag/hooks.py

```diff
@@ -598,6 +598,16 @@ def tracks_for_id(track_id):
             yield t
 
 
+def invoke_mb(call_func, *args):
+    if not config["musicbrainz"]["enabled"]:
+        return ()
+
+    try:
+        return call_func(*args)
+    except mb.MusicBrainzAPIError as exc:
+        exc.log(log)
```

Comment:
I think this might cause a crash when an exception occurs. Namely, it seems like this function needs to return an iterable, right? So I think this exception handler should probably `return ()` or similar…

Reply:
Nice catch! Yes, I'd suspect the same.

Reply:
Well caught! Fixed in e1ffadb2